api/src/services/assessmentsDummyService.ts

```diff
@@ -0,0 +1,185 @@
+import db from './db';
+
+interface ProgramParticipantsRow {
+  id: number;
+  principal_id?: number;
+  program_id?: number;
+  role_id?: number;
+}
+
+/**
+ * Return  program_participants table for any rows matching this principalId and this programId.
+ *
+ * @param {number} principalId - The given principalId
+ * @param {number} programId - The given programId
+ *
+ * @returns {ProgramParticipantsRow}
+ */
+export const insertToProgramParticipants = async (
+  principalId: number,
+  programId: number,
+  roleId: number
+): Promise<ProgramParticipantsRow> => {
+  // select from the program_participants table for any rows matching this principalId and this programId
```

Comment:
Now that we're at the PR state, you can remove these comments and any others like them from this file.